<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SEBottomUpEditorGUIClass</class>
 <widget class="QWidget" name="SEBottomUpEditorGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>239</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Structural Model:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="btnGenerate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh List</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbStructuralModel"/>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QCheckBox" name="chkPreview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Parameters</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>351</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Minimum Cut-off</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Maximum Cut-off</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="sldMin">
           <property name="maximum">
            <number>40</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Paired bases max. angle</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="spnMin">
           <property name="maximum">
            <double>40.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="sldAngle">
           <property name="maximum">
            <number>900</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="sldMax">
           <property name="maximum">
            <number>40</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="spnMax">
           <property name="maximum">
            <double>40.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="spnAngle"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>nm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>nm</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>deg</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnGenerate</sender>
   <signal>released()</signal>
   <receiver>SEBottomUpEditorGUIClass</receiver>
   <slot>onGenerateModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRefresh</sender>
   <signal>released()</signal>
   <receiver>SEBottomUpEditorGUIClass</receiver>
   <slot>onRefreshPartList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbStructuralModel</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SEBottomUpEditorGUIClass</receiver>
   <slot>onSelectNode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SEBottomUpEditorGUIClass</receiver>
   <slot>onMinSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPreview</sender>
   <signal>toggled(bool)</signal>
   <receiver>SEBottomUpEditorGUIClass</receiver>
   <slot>onPreview(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SEBottomUpEditorGUIClass</receiver>
   <slot>onMaxSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldAngle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SEBottomUpEditorGUIClass</receiver>
   <slot>onAngleSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRefreshPartList()</slot>
  <slot>onGenerateModel()</slot>
  <slot>onSelectNode(int)</slot>
  <slot>onMinSliderChanged(int)</slot>
  <slot>onMaxSliderChanged(int)</slot>
  <slot>onAngleSliderChanged(int)</slot>
  <slot>onPreview(bool)</slot>
 </slots>
</ui>
